wordpress-mobile/WordPress-iOS
# Simperium Sunset Mark III: Service Layer

### Details:

This **PR** implements _NotificationsSyncService_ + _NotificationsSyncServiceRemote_ + _RemoteNotification_, along with (hopefully) super cool unit tests

**Please** note that i had to add an extra attribute to the Core Data Model (Mark 53). It's okay to do just a simple edit, since the model 53 lives only in the `simperium-sunset` branch.

Ref #6086
### To test:
- Make sure the code is pretty enough. Key part here is: **NotificationsSyncService.sync()** method.
- Run the Unit tests, and make sure everything is smooth: `NotificationSyncServiceTests` + `NotificationSyncServiceRemoteTests` + `RemoteNotificationTests`

Needs Review: @astralbodies 
**TAG** You're it!!

cc @kwonye 

(Thanks in advance Aaron!!!)
